README.md

```diff
@@ -1,10 +1,15 @@
+
+Introduction to building packages for Nobara:
+---
+
 This repo contains the various .spec sheets, patches, and configurations Nobara applies to the modified packages it provides. You can use the config files in the mock folder to build the packages using mock.
 
 Package building basics:
+---
 
 First, we need to create a .src.rpm from the .spec sheet and patches/files it uses. This will require some tools:
 
-`sudo dnf install spectool fedpkg`
+`sudo dnf install fedpkg`
```

Comment:
spectool is needed for downloading sources, keep this.

Reply:
in my own testing, it gets brought in via fedpkg rather than as a standalone package

Reply:
double checked. it's a part of rpmdevtools which fedpkg has listed as a dependency:
<img width="641" alt="image" src="https://github.com/Nobara-Project/rpm-sources/assets/136293710/1eaa530d-2f57-47a5-a70c-dcbf051dda2a">